guides/doc-Monitoring_Project/master.adoc

```diff
@@ -0,0 +1,45 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+= {MonitoringDocTitle}
+:monitoring-project:
+:context: monitoring
+
+include::topics/con_metrics-overview.adoc[leveloffset=+1]
```

Comment:
why tho? nothing in the content is in any way "common"

Reply:
That's just how a modular repo is structured. All module files go into common/modules/ so that they can be reused in any of the other guides if needed. Another way to think about it is that common/modules/ is a sort of a database of all content from which individual guides are built.